tensorboard/plugins/interactive_inference/witwidget/notebook/colab/wit.py

```diff
@@ -179,29 +184,43 @@ def __init__(self, config_builder, height=1000):
       config_builder: WitConfigBuilder object containing settings for WIT.
       height: Optional height in pixels for WIT to occupy. Defaults to 1000.
     """
+    self._ctor_complete = False
     base.WitWidgetBase.__init__(self, config_builder)
     # Add this instance to the static instance list.
     WitWidget.widgets.append(self)
 
     # Display WIT Polymer element.
     display.display(display.HTML(self._get_element_html()))
     display.display(display.HTML(
-      WIT_HTML.format(
-        examples=json.dumps(self.examples), height=height, id=WitWidget.index)))
+      WIT_HTML.format(height=height, id=WitWidget.index)))
 
     # Increment the static instance WitWidget index counter
     WitWidget.index += 1
 
     # Send the provided config and examples to JS
     output.eval_js("""configCallback('{config}')""".format(
       config=json.dumps(self.config)))
-    output.eval_js('updateExamplesCallback()')
+    output.eval_js("""updateExamplesCallback({examples})""".format(
+      examples=json.dumps(self.examples)))
     self._generate_sprite()
+    self._ctor_complete = True
 
   def _get_element_html(self):
     return """
       <link rel="import" href="/nbextensions/wit-widget/wit_jupyter.html">"""
 
+  def set_examples(self, examples):
```

Comment:
I think we need to keep the wit id in self at init so that this method can call the correct broadcast channel.

Reply:
fixed